src/main/java/org/generationcp/middleware/service/impl/inventory/PlantingServiceImpl.java

```diff
@@ -237,7 +237,6 @@ public void savePlanting(final Integer userId, final Integer studyId, final Inte
 			.getTransactionsByStudyId(studyId, transactionStatus, ExperimentTransactionType.PLANTING);
 	}
 
-	@Override
```

Comment:
Removing this is not part of branch work?

Reply:
I may have accidentally removed it. I added it back.